server/integration/testsuite/src/test/java/org/infinispan/server/test/cs/jdbc/SingleNodeJdbcStoreIT.java

```diff
@@ -414,9 +422,9 @@ public boolean isSatisfied() throws Exception {
     public String getStoredKey(RemoteCache cache, String key) throws IOException, InterruptedException {
         // 1. marshall the key
         // 2. encode it with base64 (that's what DefaultTwoWayKey2StringMapper does)
-        // 3. prefix it with 9 (again, done by DefaultTwoWayKey2StringMapper to mark the key as byte array type)
+        // 3. prefix it with 8 (again, done by DefaultTwoWayKey2StringMapper to mark the key as wrapped byte array type)
         // 4. prefix it with UTF-16 BOM (that is what DefaultTwoWayKey2StringMapper does for non string values)
-        return '\uFEFF' + "9" + Base64.encodeBytes(cache.getRemoteCacheManager().getMarshaller().objectToByteBuffer(key));
+        return '\uFEFF' + "8" + Base64.encodeBytes(cache.getRemoteCacheManager().getMarshaller().objectToByteBuffer(key));
```

Comment:
Hrmm I am not exactly sure the different ways users would load the data, but the Key2StringMapper can still read the value out.

However if you just upgraded the server only I don't think you could load the previous table as it is stored as a byte[].  I can unwrap the WBA and store it as a byte[] still for the JDBC classes Although this will be an issue for all of the Cache Stores not just JDBC.  WDYT?